cypress/plugins/webpack.config.js

```diff
@@ -11,6 +11,8 @@
 const path = require('path');
 const webpack = require('webpack');
 
+const THEME_IMPORT = `'../../dist/eui_theme_amsterdam_${process.env.THEME ?? 'dark'}.css'`;
```

Comment:
Since at least 3 of us mentioned that we prefer light mode to dark, can we actually default to light mode instead? 🙈 HAHA sorry

Reply:
One other interesting option as well would be to let users completely specify legacy vs amsterdam as well. e.g.

```suggestion
const THEME_IMPORT = `'../../dist/eui_theme_${process.env.THEME ?? 'amsterdam_light'}.css'`;
```

usage:

```sh
yarn test-cypress --theme=amsterdam_dark
yarn test-cypress --theme=legacy_light
```

Honestly my best guess is that the primary folks running Cypress likely won't be changing this frequently in any case, so the slight extra verbosity doesn't bother me

Reply:
With legacy moving to a different dist folder as part of another PR, I'd prefer holding off on trying to support that layer until that merges.

Default changes to light theme.

Reply:
> With legacy moving to a different dist folder as part of another PR, I'd prefer holding off on trying to support that layer until that merges.

With this in mind, I'd suggest having 2 flags: `colormode` and `theme`, similar to how `EuiThemeProvider` works. `theme` could be added in a later PR and this PR can implement `colormode=light/dark`

Reply:
Do we need this `??` fallback if a default is already set in line 14 of `scripts/cypress.js`?

```suggestion
const THEME_IMPORT = `'../../dist/eui_theme_amsterdam_${process.env.THEME}.css'`;
```